docs/writing-policies/raw-policies.md

```diff
@@ -0,0 +1,174 @@
+docker volume create --driver local \
+                --opt type=tmpfs \
+                --opt device=tmpfs \
+                --opt o=uid=65533 \
+                policy-store
+
+# Start the policy server
+docker run --rm -it \
+    -p 3000:3000 \
+    -v $(pwd)/policies.yml:/policies.yml \
+    -v policy-store:/registry \
+    ghcr.io/kubewarden/policy-server:1.9.0 \
+    --ignore-kubernetes-connection-failure=true
+```
+
+NOTE: the flag `--ignore-kubernetes-connection-failure=true` is required to start the policy server without Kubernetes.
+However, it is possible to start the Policy Server with/inside Kubernetes and use the raw validation endpoint.
+Raw policies can access context-aware[ capabilities](../explanations/context-aware-policies.md) like standard policies.
+
+## Running a Policy Server inside Kubernetes without the Kubewarden controller
+
+Alternatively, the Policy Server can be deployed in Kubernetes without the Kubewarden controller.
+
+Create a `policy-server.yaml` file with the following content:
+
+```yaml
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: policy-server-deployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: policy-server
+  template:
+    metadata:
+      labels:
+        app: policy-server
+    spec:
+      containers:
+        - name: policy-server
+          image: ghcr.io/kubewarden/policy-server:1.9.0
+          ports:
+            - containerPort: 3000
+          volumeMounts:
+            - name: policy-store
+              mountPath: /registry
+            - name: policies-config
+              mountPath: /policies.yml
+              subPath: policies.yml
+      volumes:
+        - name: policy-store
+          emptyDir: {}
+        - name: policies-config
+          configMap:
+            name: policies-configmap
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: policy-server-service
+spec:
+  selector:
+    app: policy-server
+  ports:
+    - protocol: TCP
+      port: 3000
+      targetPort: 3000
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: policies-configmap
+data:
+  policies.yml: |
+    raw-mutation:
+      url: ghcr.io/kubewarden/tests/raw-mutation-policy:v0.1.0
+      allowedToMutate: true
+      settings: {}
+```
+
+Apply the configuration:
+
+```bash
+kubectl apply -f policy-server.yaml
+```
+
+## Writing raw policies
+
+Similarly to policies that validate Kubernetes resources, raw policies are written in WebAssembly using Kubewarden SDKs.
+Please refer to language-specific documentation for more information:
+
+- [Go](../writing-policies/go/10-writing-raw-policies.md)
+
+## Metadata
+
+Policies that validate arbitrary JSON documents must have the `policyType` field set to `raw` in the metadata section:
+
+```yaml
+[...]
+mutating: true
+executionMode: kubewarden-wapc
+policyType: raw
+[...]
+```
+
+Please refer to the [metadata](metadata.md) section for more information about policy metadata.
+
+## Using the raw validation endpoint
```

Comment:
I think this section should not be here, but directly in the "specification" section.